apps/mobile-alphabet/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Is the read \ write permission absolutely necessary given that the audio is currently remote?